src/Sarif.Viewer.VisualStudio.UnitTests/Sarif.Viewer.VisualStudio.UnitTests.csproj

```diff
@@ -89,8 +89,9 @@
       <Name>Sarif.Viewer.VisualStudio</Name>
     </ProjectReference>
     <ProjectReference Include="..\Sarif\Sarif.csproj">
-      <Project>{CC9B247E-7103-4BF7-BDAB-6E625B3680A8}</Project>
+      <Project>{c5fc2683-47cf-447f-b5ea-9fde0fd78754}</Project>
       <Name>Sarif</Name>
+      <AdditionalProperties>TargetFramework=net452</AdditionalProperties>
```

Comment:
Since this project wasn't replaced with a NET Core project, specify which version output of Sarif it should use.